src/Domain/Entities/Categories.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities
+{
+    public class Categories
```

Comment:
Dlaczego `Categories`, a nie `Category` ? Jeśli klasa nie zachowuje się jak kolekcja, tylko jak obiekt, to używaj rzeczownika w liczbie pojedynczej